src/components/Security/Roles/Page.vue

```diff
@@ -5,6 +5,14 @@
         <headline>Roles</headline>
       </b-col>
       <b-col class="text-right mt-3">
+        <b-button
+          v-if="displayRevokeAnonymous"
```

Comment:
```suggestion
          :disabled="!displayRevokeAnonymous"
          :title="displayRevokeAnonymous ? 'Reduce anonymous rights to the minimum' : 'You cannot revoke anonymous rights either because you don\'t have the permissions to do it or because there is no administrator user yet in the system.'
```